source/common/network/socket_option_impl.cc

```diff
@@ -12,10 +12,11 @@
 namespace Envoy {
 namespace Network {
 
+using ::envoy::config::core::v3::SocketOption;
```

Comment:
Don't do that; it's generally not the coding style in envoy

Reply:
Removed. Sorry about that.